src/overlays/actors/ovl_En_Bee/z_en_bee.c

```diff
@@ -118,121 +118,122 @@ void EnBee_Destroy(Actor* thisx, PlayState* play) {
     Collider_DestroyCylinder(play, &this->collider);
 }
 
-void func_80B5A854(EnBee* this) {
+void EnBee_SetupFlyIdle(EnBee* this) {
     s32 pad;
-    Vec3f sp48;
-    s16 sp46;
+    Vec3f tmpPos;
```

Comment:
```suggestion
    Vec3f tempPos;
```
No need to abbreviate words this short.

Reply:
d699412